store/imagestore/migrate_test.go

```diff
@@ -871,7 +945,7 @@ func populateDBV6(db *db.DB, dbVersion int, aciInfos []*ACIInfoV6, remotes []*Re
 
 	fn = func(tx *sql.Tx) error {
 		for _, aciinfo := range aciInfos {
-			_, err := tx.Exec("INSERT INTO aciinfo (blobkey, name, importtime, lastused, latest, size, treestoresize, souceurl, insecureoptions) VALUES ($1, $2, $3, $4, $5, $6, $7, $8)", aciinfo.BlobKey, aciinfo.Name, aciinfo.ImportTime, aciinfo.LastUsed, aciinfo.Latest, aciinfo.Size, aciinfo.TreeStoreSize, aciinfo.InsecureOptions)
```

Comment:
just noticed this "sourceurl" now. I thought that the query should error due to wrong number of elements (9 != 8)...